README.md

```diff
@@ -50,6 +50,17 @@ optional arguments:
   -o [OUTPUT], --output [OUTOUT] directory and name of output folder at, if not specified, use current folder
 
 
+The sra metadata of specified accessions can be downloaded by another script:
+
+download_sra_metadata_by_accessions.py [-a ACCESSION] [-o [OUTPUT]]
+
+The script can process multiple accessions at once.
```

Comment:
Could you add something like 'Use spaces between each accession and output file'?

Reply:
@ZhiXuanLai